src/js/stores/MesosSummaryStore.js

```diff
@@ -38,12 +37,7 @@ function startPolling() {
       } else {
         // If not active, push null placeholder. This will ensure we maintain
         // history when navigating back, for case where history server is down.
-
-        // Use {silent: true} Because we only want to push a summary on the stack without side
-        // effects (like re-rendering etc). The tab is out of focus so we
-        // don't want it to do any work. It only matters that there is
-        // appropriate history when we return focus to the tab.
-        MesosSummaryStore.processSummaryError({silent: true});
+        MesosSummaryActions.fetchSummary(undefined, true);
```

Comment:
Can we instead just do this at the bottom of the file: `MesosSummaryStoreInstance = new MesosSummaryStore();`
and call `MesosSummaryStoreInstance.processSummaryError({silent: true});` just like we were doing before?

I don't like the idea of calling a method on an Actions file that we know will return faked data as if it were making a call to an API. I don't believe that's the best layer to fake the data.